pkg/registry/policy/poddisruptionbudget/strategy.go

```diff
@@ -109,7 +111,8 @@ func (podDisruptionBudgetStatusStrategy) PrepareForUpdate(ctx context.Context, o
 
 // ValidateUpdate is the default update validation for an end user updating status
 func (podDisruptionBudgetStatusStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
-	// TODO: Validate status updates.
```

Comment:
flip this around to return early for policy/v1beta1 for backwards compatibility, but to validate in all other cases. that ensures we continue validating if future versions are introduced (v2alpha1, etc)

Reply:
also, add a unit test that makes sure this does not call validation for `policy/v1beta1`, does call it for `policy/v1` and `policy/random-version`

Reply:
I have fixed the logic in the `ValidateUpdate` method and added a unit test.

Reply:
also compare `policyapiv1beta1.SchemeGroupVersion.Group == requestInfo.APIGroup`